content/administration/maintain/hosting_changes.rst

```diff
@@ -48,13 +48,34 @@ From Odoo Online to Odoo.sh
 From Odoo.sh to Odoo Online
 ===========================
 
-.. warning:: Odoo Online is not compatible with **non-standard apps**.
+#.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your
+    production build.
+#.  **If you have time constraints, contact us earlier to schedule the transfer.**
```

Comment:
+ indentation problem

```suggestion
#. **If you have time constraints, contact us earlier to schedule the transfer.**
```

Also, I wouldn't put this as a mandatory step, but in the note instead.

>>>> I guess you just put it this way to stay consistent with the top of the page, though, so maybe just leave it, or change the whole page MOUHAHAHAHHAHA